crates/ruff_python_formatter/src/other/keyword.rs

```diff
@@ -13,10 +13,11 @@ impl FormatNodeRule<Keyword> for FormatKeyword {
             arg,
             value,
         } = item;
-        if let Some(argument) = arg {
-            write!(f, [argument.format(), text("=")])?;
+        if let Some(arg) = arg {
+            write!(f, [arg.format(), text("="), value.format()])
+        } else {
+            // TODO(konstin): dangling comments after the stars?
```

Comment:
I don't think that we have to preserve these. It seems a rather weird place to put comments.